tr/Cargo.toml

```diff
@@ -18,3 +18,10 @@ default = ["gettext-rs"]
 lazy_static = "1.2"
 gettext-rs = { version = "0.7", optional = true, features = ["gettext-system"] }
 gettext = { version = "0.4", optional = true }
+
+[dev-dependencies]
+criterion = "*"
```

Comment:
I think we need an actual version there.

```suggestion
criterion = "0.5"
```

Reply:
I forgot that! Fixed.